src/main/modules/dashboard/consts.jsx

```diff
@@ -0,0 +1,98 @@
+export const expenses = [
```

Comment:
if we are not using this constant anymore remove this

Reply:
added to the Expense form